contributing.md

```diff
@@ -35,25 +35,16 @@ yarn testonly --testPathPattern "production" -t "should allow etag header suppor
 
 ## Testing in your own app
 
-First `next-server` needs to be linked:
+Because of the way Node.js resolves modules the easiest way to test your own application is copying it into the `test` directory.
 
 ```
-cd packages/next-server
-npm link
+cp -r yourapp <next.js directory>/test/integration/yourapp
 ```
 
-Then `next` needs to link to `next-server`, and be linked itself:
+Make sure you remove `react` `react-dom` and `next` from `test/integration/yourapp/node_modules` as otherwise they will be overwritten.
```

Comment:
`````suggestion
Make sure you remove `react` `react-dom` and `next` from `test/integration/yourapp/node_modules` as otherwise they will be overwritten.

```bash
rm -rf <next.js directory>/test/integration/yourapp/{react,react-dom,next,next-server}
```
`````